packages/e2e-tests/src/specs/editor/publishingFlow.js

```diff
@@ -110,7 +110,9 @@ describe('Publishing Flow', () => {
     expect(postPermalink).not.toBeNull();
     expect(postPermalink).toStrictEqual(expect.any(String));
 
-    await Promise.all([page.goto(postPermalink), page.waitForNavigation()]);
+    await page.goto(postPermalink, {
+      waitUntil: 'networkidle0',
+    });
```

Comment:
Would the case be similar here: https://github.com/google/web-stories-wp/blob/a640f7e734ec377c79d7295846d839991d0d5887/packages/e2e-tests/src/specs/wordpress/quickEdit.js#L77-L80

That it should first wait for the click and only then `waitForNavigation()`?

Reply:
@spacedmonkey Could you take a look at all instances of these cases? Would be good to have uniform handling of page navigation in e2e tests.

Reply:
I did. The issue with this test is, `page.waitForNavigation()` when you click / interact with an element and the browser changes url, but just does do anything with a togo. In this PR, I replace all the places that do this, with a wait until network connections have stopped for 500ms.